client/components/jetpack/card/jetpack-product-card/index.tsx

```diff
@@ -4,6 +4,7 @@
 import classNames from 'classnames';
 import { useTranslate, TranslateResult } from 'i18n-calypso';
 import { isFinite, isNumber } from 'lodash';
+import moment, { Moment } from 'moment';
```

Comment:
As a rule, the code should not import `moment` directly but use the `useLocalizedMoment` hook. That means instead of Moment being imported in this JS bundle it's being loaded once and provided via Provider. See the [`localized-moment` component](https://wpcalypso.wordpress.com/devdocs/client/components/localized-moment/README.md) for details!

Reply:
Thanks for the heads up!